gluten-ut/spark32/src/test/scala/org/apache/spark/sql/execution/GlutenSQLWindowFunctionSuite.scala

```diff
@@ -40,7 +39,7 @@ class GlutenSQLWindowFunctionSuite extends SQLWindowFunctionSuite with GlutenSQL
     Row(4953, 10, decimal(6037.28)),
     Row(35403, 5, decimal(6034.70)),
     Row(35803, 12, decimal(5284.87)),
-    Row(60865, 5, decimal(-227.83)),
+    Row(60865, 5, decimal(-227.82)),
```

Comment:
The modification here is because the original input resulted in an output of Velox being 4464.706667, while ClickHouse was 4464.706666, which should be due to the difference in backend calculation precision and is unrelated to the functionality being tested here. After adjusting the input, the results from Velox and ClickHouse can be made consistent.